xen/arch/x86/tpm.c

```diff
@@ -933,6 +925,7 @@ void tpm_hash_extend(unsigned loc, unsigned pcr, uint8_t *buf, unsigned size,
     uint32_t evt_log_size;
 
     find_evt_log(&evt_log_addr, &evt_log_size);
+    evt_log_addr = __va(evt_log_addr);
 
 #ifndef __EARLY_TPM__
```

Comment:
Mapping protected by this should be dropped, event log is already mapped and this results in exception because region is not aligned properly. Same goes for unmapping at the end of the function.

Reply:
See https://github.com/TrenchBoot/xen/compare/cc6e675973ecc45d3e99d0c9b1301f16d3912476..f74bb7a7e506d9beb2e8267a9d3b4572b968b62d#diff-5861d423276bd98c31fb2a501569110297cbe6488842e0359435a3af795c24aeL930-L971